lib/simulator-xcode-6.js

```diff
@@ -571,6 +574,18 @@ class SimulatorXcode6 extends EventEmitter {
     }
   }
 
+  async enableCalendarAccess (bundleID) {
+    await this.calendar.enableCalendarAccess(bundleID);
+  }
+
+  async disableCalendarAccess (bundleID) {
+    await this.calendar.disableCalendarAccess(bundleID);
+  }
+
+  async hasCalendarAccess (bundleID) {
+    return this.calendar.hasCalendarAccess(bundleID);
```

Comment:
nitpick:

even though these two lines are functionally equivalent:

```js
return this.calendar.hasCalendarAccess(bundleId);
return await this.calendar.hasCalendarAccess(bundleId);
```

we prefer the latter in our code because it signals to the reader that we are actually calling an async function, which you might not know otherwise, and be tempted to do something like:

```js
this.calendar.hasCalendarAccess(bundleID);
```

which is NOT the same as

```js
await this.calendar.hasCalendarAccess(bundleID);
```